tests/jax/core_system_components/launcher_test.py

```diff
@@ -102,11 +102,10 @@ def test_add_multi_process(mock_builder: MockBuilder) -> None:
 
 
 def test_add_multi_process_two_add_calls(mock_builder: MockBuilder) -> None:
-    """Test calling add more than one time method in the Launcher for the case of multi process
+    """Test calling add more than one time method in the Launcher for the case of multi process # noqa:E501
```

Comment:
Do we need this ignore here? Should we perhaps just make the docstring shorter? Also saw this in other sections of the code. I think it would be better to shorten the string or use new lines etc.

Reply:
Agreed, I didn't write the docstrings, just added the noqa. Let's create another ticket to improve our docstrings.